pytorch/inference/docker/1.10/py3/sdk1.19.0/Dockerfile.neuron

```diff
@@ -12,7 +12,7 @@ ARG MAMBA_VERSION=4.12.0-0
 
 # See http://bugs.python.org/issue19846
 ENV LANG C.UTF-8
-ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /lib/x86_64-linux-gnu:/opt/conda/lib/:$LD_LIBRARY_PATH
```

Comment:
Why is this change necessary? Was this copied over from a different dockerfile, or was it obtained from a source somewhere online? If it is from an online source, could you please paste a reference to that here, so that we know why this change is made for future reference?

Reply:
Synced offline with @kevinyang8. This change is a fix for a known issue within the Neuron DLC specifically. Resolving this conversation.